src/pykeen/training/training_loop.py

```diff
@@ -974,3 +983,30 @@ def _load_state(self, path: Union[str, pathlib.Path]) -> None:
         np.random.set_state(checkpoint['np_random_state'])
         torch.random.set_rng_state(checkpoint['torch_random_state'])
         logger.info(f"=> loaded checkpoint '{path}' stopped after having finished epoch {checkpoint['epoch']}")
+
+
+class AcceleratedTrainingLoop(TrainingLoop, ABC):
+    """A training loop with HF accelerate."""
+
+    def __init__(self, **kwargs) -> None:
+        """Initialize the training loop mixin."""
+        import accelerate
```

Comment:
If `accelerate` is an optional dependency (which I support), we should wrap this into some code which generate a user-friendly error message with hints how to install the required optional dependency.

Reply:
good point. We could even consider making this part of the normal training loop that can be optionally enabled